org.freedesktop.Sdk.Extension.mingw-w64.metainfo.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="runtime">
+  <id>org.freedesktop.Sdk.Extension.mingw-w64</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0 and LGPL-2.0-or-later and ZPL-2.1 and Public Domain</project_license>
```

Comment:
appstream-util doesn't like the `Public Domain` licence. I'm not sure what to put here - MinGW toolchain consists of multiple components under different licences.